RecipeMatch/View Controllers/StreamViewController.h

```diff
@@ -10,11 +10,13 @@
 #import "DraggableViewBackground.h"
 
 NS_ASSUME_NONNULL_BEGIN
-
-@interface StreamViewController : UIViewController <PreferencesViewControllerDelegate>
-@property NSMutableArray *recipes;
-
+@interface StreamViewController : UIViewController <PreferencesViewControllerDelegate, DraggableViewBackgroundDelegate>
+- (void)checkLikeStatusFromDraggableViewBackground:(DraggableView *_Nullable)nextCard withCompletion:(void (^_Nullable)(BOOL liked, NSError *_Nullable error))completion;
+- (void)checkSaveStatusFromDraggableViewBackground:(DraggableView *_Nullable)card withCompletion:(void (^_Nullable)(BOOL liked, NSError *_Nullable error))completion;
+- (void)postSavedRecipeFromDraggableViewBackgroundWithId:(NSString * _Nullable)recipeId title:( NSString * _Nullable )title image: (NSString * _Nullable)image andCompletion:(void (^_Nullable)(BOOL succeeded, NSError * _Nullable error))completion;
+- (void)postLikedRecipeFromDraggableViewBackgroundWithId:(NSString * _Nullable)recipeId recipeTitle:(NSString * _Nullable)title image: (NSString * _Nullable)image andCompletion:(void (^_Nullable)(BOOL succeeded, NSError * _Nullable error))completion;
+- (void)unlikeRecipeFromDraggableViewBackgroundWithId:(NSString * _Nullable)recipeId andCompletion: (void (^_Nullable)(BOOL succeeded, NSError *_Nullable error))completion;
+- (void)countLikesFromDraggableViewBackgroundWithId:(NSString * _Nullable)recipeId andCompletion: (void (^_Nullable)(int likes, NSError * _Nullable error))completion;
```

Comment:
these all are protocol methods defined in `DraggableViewBackgroundDelegate`, right? 
so no need to define them here as well